File: project/recipio/ios/Recipio/Utilities/NativeUtils.swift
import Foundation

/// Small helpers for native iOS code (bridges, future modules).
enum NativeUtils {
  static var isDebugBuild: Bool {
#if DEBUG
    return true
#else
    return false
#endif
  }

  static var bundleIdentifier: String {
    Bundle.main.bundleIdentifier ?? ""
  }
}

File: project/recipio/android/app/src/main/java/com/masterfabric/recipio/util/NativeUtils.kt
package com.masterfabric.recipio.util

import com.masterfabric.recipio.BuildConfig

/**
 * Small helpers for native Android code (bridges, future modules).
 */
object NativeUtils {

  fun isDebugBuild(): Boolean = BuildConfig.DEBUG

  fun applicationId(): String = BuildConfig.APPLICATION_ID

  fun buildType(): String = BuildConfig.BUILD_TYPE
}
